pkg/mcproducer/testdata/machineconfig-test.yaml

```diff
@@ -32,8 +32,9 @@ spec:
         - contents: |
             [Unit]
             Description=Pull oot kernel module image
```

Comment:
Should have different description for both services?

Reply:
fixed